src/main/java/controller/BlackjackController.java

```diff
@@ -0,0 +1,128 @@
+package controller;
+
+import domain.CardDeck;
+import domain.GameManger;
+import domain.GameResult;
+import domain.TrumpCard;
+import domain.user.Dealer;
+import domain.user.Player;
+import domain.user.User;
+import java.util.List;
+import java.util.Map;
+import view.CardConverter;
+import view.InputView;
+import view.OutputView;
+import view.Parser;
+
+public class BlackjackController {
+    private final OutputView outputView;
+    private final InputView inputView;
+
+    public BlackjackController(OutputView outputView, InputView inputView) {
+        this.outputView = outputView;
+        this.inputView = inputView;
+    }
+
+    public void run() {
+        List<String> playerNames = Parser.parseStringToList(inputView.inputUsers());
+        List<Player> users = playerNames.stream().map(Player::new).toList();
+        GameManger gameManger = new GameManger(users, new Dealer(), new CardDeck());
+        distributionFirstCard(gameManger, playerNames);
+
+        additionalPlayerCard(gameManger, playerNames);
+        additionalDealerCard(gameManger);
+
+        createGameResult(gameManger, playerNames);
+        calculateGameResult(gameManger);
+
+        inputView.close();
+    }
+
+    private void distributionFirstCard(GameManger gameManger, List<String> playerNames) {
+        gameManger.firstHandOutCard();
+
+        displayOpenCard(gameManger.getDealer().getName(), gameManger.getDealer());
+        playerNames.forEach(playerName -> displayOpenCard(playerName, gameManger.findUserByUsername(playerName)));
+    }
+
+    private void additionalDealerCard(GameManger gameManger) {
+        User dealer = gameManger.getDealer();
+        while (!dealer.isImpossibleDraw()) {
+            dealer.drawCard(gameManger.handOutCard());
+            outputView.displayDealerAddCard();
+        }
+    }
+
+    private void additionalPlayerCard(GameManger gameManger, List<String> playerNames) {
+        playerNames.forEach(playerName ->
+                addCardAllPlayer(gameManger, playerName));
+    }
+
+    private void calculateDealerResult(Map<User, GameResult> gameResult) {
+        int dealerLoseCount = getResultStateCount(gameResult, GameResult.WIN);
+        int dealerWinCount = getResultStateCount(gameResult, GameResult.LOSE);
+        int dealerDrawCount = getResultStateCount(gameResult, GameResult.DRAW);
+
+        outputView.displayDealerGameResult(dealerLoseCount, dealerWinCount, dealerDrawCount);
+    }
```

Comment:
Long -> int 변환 때문에 조금 아쉽지만... 아래와 같이 변경해볼 수도 있을 것 같아요. 

```suggestion
    private void calculateDealerResult(Map<User, GameResult> gameResult) {
        Map<GameResult, Long> dealerCounts = gameResult.values()
            .stream()
            .collect(Collectors.groupingBy(it -> it, Collectors.counting()));

        outputView.displayDealerGameResult(
            Math.toIntExact(dealerCounts.get(GameResult.WIN)),
            Math.toIntExact(dealerCounts.get(GameResult.LOSE)),
            Math.toIntExact(dealerCounts.get(GameResult.DRAW))
        );
    }
```

혹은 `dealerCounts`를 view에 넘겨주고,  view에서 원하는 순서로 출력하도록해도 될 것 같아요. 😃

Reply:
아... 현재 User 들의 결과를 가지고 반대로 출력하여 Dealer의 승패를 판단하고있는거군요? 🤔 
딜러의 승패 결과에 대한 계산(= 도메인 로직)을 컨트롤러에서 수행하고있는 것 같은데요. 👀

이 부분은 GameManager에게 Dealer의 승패 결과를 계산해달라고 메시지를 보낸 결과로 `dealerCounts`를 얻도록 해볼 수는 없을까요?